packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -0,0 +1,187 @@
+    --_iui-expandable-block-header-focus-thickness: 2px;
+
+    &:where(.iui-expanded) {
+      --_iui-expandable-block-header-bottom-border-radii: 0;
+    }
+  }
+  @if $variant == 'borderless' {
+    --_iui-expandable-block-border: none;
+    --_iui-expandable-block-background-color: transparent;
+    --_iui-expandable-block-content-padding: none;
+    --_iui-expandable-block-header-background-color-disabled: transparent;
+    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-transparent-hover);
+    --_iui-expandable-block-header-focus-offset: -1px;
+    --_iui-expandable-block-header-focus-thickness: 1px;
+
+    &:where(.iui-expanded) {
+      --_iui-expandable-block-header-bottom-border-radii: inherit;
+    }
+  }
+}
+
+/// @arg $size Can be one of: default, small
+@mixin iui-expandable-block-size($size: 'default') {
+  @if $size == 'default' {
+    --_iui-expandable-block-expander-icon-size: var(--iui-size-m);
+    --_iui-expandable-block-header-gap: var(--iui-size-s);
+    --_iui-expandable-block-header-padding: var(--iui-size-s);
+  }
+  @if $size == 'small' {
+    --_iui-expandable-block-expander-icon-size: var(--iui-size-s);
+    --_iui-expandable-block-header-gap: var(--iui-size-xs);
+    --_iui-expandable-block-header-padding: 6px var(--iui-size-xs);
+  }
+}
+
+@mixin iui-expandable-block-header {
+  @include iui-reset;
+  @include iui-focus(
+    $offset: var(--_iui-expandable-block-header-focus-offset),
+    $thickness: var(--_iui-expandable-block-header-focus-thickness)
+  );
+  display: flex;
+  gap: var(--_iui-expandable-block-header-gap);
+  align-items: center;
+  padding: var(--_iui-expandable-block-header-padding);
+  cursor: pointer;
+  user-select: none;
+  color: var(--iui-color-text);
+  line-height: inherit;
+  font-family: inherit;
+  text-align: left;
+  background-color: var(--_iui-expandable-block-background-color);
+  border: var(--_iui-expandable-block-border);
+  border-radius: inherit;
+  border-bottom-left-radius: var(--_iui-expandable-block-header-bottom-border-radii, inherit);
+  border-bottom-right-radius: var(--_iui-expandable-block-header-bottom-border-radii, inherit);
+  transition: background-color var(--iui-duration-1) ease-out;
+  @media (forced-colors: active) {
+    --_iui-expandable-block-expander-icon-fill: CanvasText;
+  }
+
+  &:where(:hover) {
+    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-hover);
+    --_iui-expandable-block-header-caption-color: var(--iui-color-text-hover);
+    --_iui-expandable-block-header-label-color: var(--iui-color-text-hover);
+
+    background-color: var(--_iui-expandable-block-header-background-color-hover);
+
+    @media (forced-colors: active) {
+      --_iui-expandable-block-header-caption-color: Highlight;
+      --_iui-expandable-block-header-label-color: Highlight;
+      --_iui-expandable-block-expander-icon-fill: Highlight;
+      border-color: Highlight;
```

Comment:
I might be wrong, but should we update `--iui-color-border` since border is set on [L81](https://github.com/iTwin/iTwinUI/pull/821/files#diff-7139d3558365a97d23934152a27c2e4c8faba3aaeb01912529f15c90f764b7a2R81) that includes the color?

Reply:
Generally we don't override variables from iTwinUI-Variables, instead we override component specific variables (theoretical example: `--_iui-expandable-block-header-border-color`).  I could introduce a component variable instead if you'd prefer.

Reply:
Ohh, I didn't realize that was a variables package var and not a local var. In that case, whatever you prefer would be good. I just wanted to confirm that since this PR is trying to add local vars for everything, maybe this could also be a variable.